tasks.py

```diff
@@ -153,7 +153,7 @@ def _bump_version(version_: Version, pre_release: bool) -> Version:
         )
     else:
         # create the first pre-release version
-        new_version = Version(f"{version_.major}.{version_.minor}.{version_.micro}.dev1")
+        new_version = Version(f"{version_.major}.{version_.minor}.{version_.micro + 1}.dev0")
```

Comment:
Question: why do we call the third version a `micro`? Generally I have heard semver referenced as `major.minor.match`

Reply:
This isn't our code.
It's coming from `Version` which handles other types of versioning formats.
https://packaging.pypa.io/en/stable/version.html#packaging.version.Version